jujugui/static/gui/src/app/components/entity-header/test-entity-header.js

```diff
@@ -57,6 +57,7 @@ describe('EntityHeader', function() {
                  'rendered downloads does not match entity downloads');
   });
 
+  /* Test will need to be moved to test-entity-content.js
```

Comment:
Can you just delete this? When the tags are done in entity content it'll be a different test entirely